README.md

```diff
@@ -26,6 +26,107 @@ the repo may contact me at blueness@gentoo.org.
 
 ==========
 
+BUILD INSTRUCTIONS
```

Comment:
I see no point in duplicating the text in the README, please drop the changes to README.md, we have to rewrite it anyways and then we will include a link (or reference) to BUILD. As noted by Arsen, please amend your commit messages, e.g. (to keep it simple it is OK to force push your branch instead of opening another PR, but both are OK, depending which one is easier for you):

Add build instructions

- document how to install all dependencies
- document how to get source via tarball or git
- document how to build eudev

Reply:
I've backed the changed out of the readme in my branch. And updated the commit message.

I hope this doesn't come across as being difficult or combative, but I want to share my "new person coming into this project" first impressions. I feel like we're going in circles here. Over the past two weeks I've been asked to do the following with the documentation:

Place build/install instructions in a file called BUILD, exclusively, don't call it INSTALL.
Put the instructions in the README file, dropping BUILD.
Put a summary of instructions in README with full instructions in BUILD.
Call the file with build/install instructions INSTALL, not BUILD.
Keep the instructions in BUILD, drop everything from README.

It seems each existing member of this project has a different view of how adding new documentation should be handled. And that's okay, there is bound to be some stylistic differences. But for a new person coming in, this is a bit confusing to get such a wide range of contradicting instructions. Getting this one page of documentation committed has been a two week process of minor edits, file renames, and reverting edits.

I feel this process could be helped by having someone designated or declared as being the mentor/leader/final-say person for new contributors. Maybe have someone listed in the CONTRIBUTING file as a contact point to act as the mentor/go-between?

Maybe this isn't the right place to have this conversation, but it seems most people in this group aren't on the debian-init-diversity mailing list where I got introduced to the project so I'm not sure how best to address concerns like this. But it's something I feel we could collectively work to improve for future commits and contributors.